tools/wlr-assets/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+pub mod bld_cfg;
+
+use std::error::Error;
+use std::fmt::{Display, Formatter, Result as FmtResult};
+use std::path::Path;
+
+type BoxedError = Box<dyn Error>;
+
+pub fn download_asset(asset_url: &str, target_folder: &Path) -> Result<(), BoxedError> {
```

Comment:
I see this method is duplicated on `python/tools/wlr-libpython/build.rs`. Can we reuse the methods from this crate there?

Reply:
This is my mistake. I split `wlr-libpython` into the `wlr-assets` and `wlr-libpy` and was certain that I have deleted it, but it seems I have not. Just fixed that.